README.md

```diff
@@ -163,6 +163,9 @@ For more examples check out our [LLM Inference Guide](https://docs.openvino.ai/2
 ```sh
 #Download and convert to OpenVINO dreamlike-anime-1.0 model
 optimum-cli export openvino --model dreamlike-art/dreamlike-anime-1.0 --task stable-diffusion --weight-format fp16 dreamlike_anime_1_0_ov/FP16
+
+#Download, convert to OpenVINO and apply int8 hybrid quantization to dreamlike-anime-1.0 model
```

Comment:
Maybe explicitly state in the comment that this is an alternative, e.g.:
`Or use INT8 hybrid quantization to optimize and speed up the model` or
`Also you can use INT8 hybrid quantization to further optimize the model and reduce the inference latency`, etc,

Reply:
Updated